lib/kube/proxy/forwarder.go

```diff
@@ -2387,34 +2383,38 @@ type clusterSession struct {
 	// rbacSupportedResources is the list of resources that support RBAC for the
 	// current cluster.
 	rbacSupportedResources rbacSupportedResources
-	// connCtx is the context used to monitor the connection.
-	connCtx context.Context
-	// connMonitorCancel is the conn monitor connMonitorCancel function.
-	connMonitorCancel context.CancelCauseFunc
+	// sessionCtx is used with one more connection contexts.
```

Comment:
nit: Is this supposed to say:
```suggestion
	// sessionCtx is used with one or more connection contexts.
```